src/group_theory/perm/cycles.lean

```diff
@@ -129,7 +129,8 @@ noncomputable def is_cycle.gpowers_equiv_support {σ : perm α} (hσ : is_cycle
 equiv.of_bijective (λ τ, ⟨τ (classical.some hσ),
 begin
   obtain ⟨τ, n, rfl⟩ := τ,
-  rw [finset.mem_coe, coe_fn_coe_base, subtype.coe_mk, gpow_apply_mem_support, mem_support],
+  rw [finset.mem_coe, @coe_fn_coe_base _ _ (λ _, α → α), subtype.coe_mk,
```

Comment:
It seems `rw` and `simp` are not smart enough to figure out the `(λ _, α → α)` (even if it's an `out_param`). Should we add a non-dependent `coe_fn_coe_base'` where the argument would just be `α → α`?

Reply:
May be `α → β` will work as well. I don't know why `coe_fn_coe_base'` doesn't work. I'll be very glad if you'll fix it one way or another.

Reply:
I have added a non-dependent `coe_fn_coe_base'`, which works here.